openedx_proversity_reports/tasks.py

```diff
@@ -82,3 +84,25 @@ def generate_time_spent_report(courses):
     return {
         'time_spent_data': time_spent_report_data if time_spent_report_data else {}
     }
+
+
+@task(default_retry_delay=5, max_retries=5)  # pylint: disable=not-callable
+def generate_learning_tracker_report(courses, *args, **kwargs):
+    """
+    Return the time spent data for the given courses.
+
+    Args:
+        courses: Course ids list.
+    Returns:
+        Dict with the data for every course.
+    """
+    data = {}
+
+    for course in courses:
+
+        try:
+            data[course] = LearningTrackerReport(course).generate_report()
+        except InvalidKeyError:
+            continue
```

Comment:
Should we catch another Exception or maybe a broad exception? @andrey-canon 
Just not to return a 500 error.